es-ds-docs/pages/organisms/form.vue

```diff
@@ -0,0 +1,135 @@
+<script setup lang="ts">
+const inline = ref(false);
+
+const propTableRows = [
+    ['inline', 'Boolean', 'false', 'Display labels and form controls on a single horizontal row'],
+    ['novalidate', 'Boolean', 'null', 'Disables browser native HTML5 validation on controls in the form'],
```

Comment:
Since we're not doing anything special to provide this prop relative to all the other attributes a `<form>` tag can take, I might be in favor of removing this here and letting people look up [attributes of the form tag itself](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form#attributes_for_form_submission) to see what they can do.